hypershift-operator/controllers/nodepool/capi.go

```diff
@@ -67,6 +67,13 @@ func (c *CAPI) Reconcile(ctx context.Context) error {
 		return err
 	}
 
+	// Reconcile ORC resources
+	if nodePool.Spec.Platform.Type == hyperv1.OpenStackPlatform {
```

Comment:
I don't know if this pattern will be accepted. Suggestions are welcome!

Reply:
@bryan-cox I realized that we probably need to handle the OpenStack RHCOS image by Nodepool, since the release image can be overridden per Nodepool?